addons/website_sale/static/src/js/website_sale_configurators.js

```diff
@@ -50,21 +77,59 @@ WebsiteSale.include({
                     optional_products: JSON.stringify(optionalProducts.map(this._serializeProduct)),
                     ...this._getAdditionalRpcParams(),
                 });
-                if (options.goToCart) {
-                    window.location.pathname = '/shop/cart';
-                } else {
-                    wSaleUtils.updateCartNavBar(values);
-                    wSaleUtils.showCartNotification(this.call.bind(this), values.notification_info);
-                }
-                // Reload the product page after adding items to the cart. This is needed, for
-                // example, to update the available stock.
-                this._getCombinationInfo($.Event('click', { target: $('#add_to_cart') }));
+                this._onConfigured(options, values);
             },
             discard: () => {},
             ...this._getAdditionalDialogProps(),
         });
     },
 
+    /**
+     * Opens the combo configurator dialog.
+     *
+     * @param combos The combos of the product.
+     * @param remainingData Other data needed to open the combo configurator.
+     */
+    _openComboConfigurator(combos, remainingData) {
+        this.call('dialog', 'add', ComboConfiguratorDialog, {
+            combos: combos.map(combo => new ProductCombo(combo)),
+            ...remainingData,
+            date: serializeDateTime(DateTime.now()),
+            edit: false,
+            isFrontend: true,
+            save: async (comboProductData, selectedComboItems, options) => {
+                this._trackProducts([{
+                    'id': this.rootProduct.product_id,
+                    'display_name': remainingData.display_name,
+                    'category_name': remainingData.category_name,
+                    'currency_name': remainingData.currency_name,
+                    'price': comboProductData.price,
+                    'quantity': comboProductData.quantity,
+                }]);
+
+                const values = await rpc('/website_sale/combo_configurator/update_cart', {
+                    combo_product_id: this.rootProduct.product_id,
+                    quantity: comboProductData.quantity,
+                    selected_combo_items: selectedComboItems.map(this._serializeComboItem),
+                });
+                this._onConfigured(options, values);
+            },
+            discard: () => {},
+        });
+    },
+
+    _onConfigured(options, values) {
+        if (options.goToCart) {
+            window.location.pathname = '/shop/cart';
+        } else {
+            wSaleUtils.updateCartNavBar(values);
+            wSaleUtils.showCartNotification(this.call.bind(this), values.notification_info);
+        }
+        // Reload the product page after adding items to the cart. This is needed, for
+        // example, to update the available stock.
+        this._getCombinationInfo($.Event('click', { target: $('#add_to_cart') }));
```

Comment:
we're deprecating jquery (and maybe removing it in 18), please avoid using it in new code :pray:

Reply:
This is actually moved code, not new. That being said, I looked into migrating it, but it's not trivial so I'll keep this change for later (given the urgency of this PR).

Reply:
If it's only moved code, let's forget it for now indeed :+1: